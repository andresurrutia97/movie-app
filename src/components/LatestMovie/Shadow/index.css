.latest-movie__shadow {
  position: absolute;
  bottom: 0;
  margin-bottom: -5px;
  width: 100%;
  height: 100px;
  background-image: linear-gradient(to top, #000, transparent);
}
