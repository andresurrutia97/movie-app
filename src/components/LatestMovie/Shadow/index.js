import React from 'react';

import './index.css';

const index = () => <div className="latest-movie__shadow" />;

export default index;
