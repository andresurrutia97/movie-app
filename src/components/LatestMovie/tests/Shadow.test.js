import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

import Shadow from '../Shadow';

describe('<Shadow/>', () => {
  const component = <Shadow />;

  it('should render correctly ', () => {
    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should contain a <div/> tag with a className equal to "latest-movie__shadow"', () => {
    const wrapper = shallow(component);
    expect(wrapper.find('div')).toHaveLength(1);
    expect(wrapper.props().className).toBe('latest-movie__shadow');
  });
});
