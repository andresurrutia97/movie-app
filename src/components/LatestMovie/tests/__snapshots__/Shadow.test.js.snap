// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Shadow/> should render correctly  1`] = `
<div
  className="latest-movie__shadow"
/>
`;
